TimeCalc.java

```diff
@@ -1,5 +1,35 @@
 public class TimeCalc {
     public static void main(String[] args) {
-        // Replace this comment with your code
+        String time = args[0];
+        int minutesToAdd = Integer.parseInt(args[1]);
+
+        int ONE_HOUR_IN_MINUTES = 60;

```

Comment:
local variables should be camelCase. 
global (will be taught next week should be like Upper with underscores)